requestor/requirements.txt

```diff
@@ -6,3 +7,9 @@ gunicorn==20.1.0
 uvicorn==0.15.0
 
 git+https://github.com/golemfactory/yapapi-service-manager.git@b427d7946298ab0cdd01e023d81141d64619a722
```

Comment:
BTW - Do we still need a specific commit here?

Reply:
I'll check this as part of [APPS-537](https://golemproject.atlassian.net/browse/APPS-537)